internal/config/config.go

```diff
@@ -0,0 +1,210 @@
+	// FeeAccountBalanceThresholdKey is the treshold of LBTC balance (in satoshis) for the fee account, after wich we alert the operator that it cannot subsidize anymore swaps
+	FeeAccountBalanceThresholdKey = "FEE_ACCOUNT_BALANCE_THRESHOLD"
+	// TradeExpiryTimeKey is the duration in seconds of lock on unspents we reserve for accpeted trades, before eventually double spending it
+	TradeExpiryTimeKey = "TRADE_EXPIRY_TIME"
+	// TradeSatsPerByte is the sats per byte ratio to use for paying for trades' network fees
+	TradeSatsPerByte = "TRADE_SATS_PER_BYTE"
+	// PriceSlippageKey is the percentage of the slipage for accepting trades compared to current spot price
+	PriceSlippageKey = "PRICE_SLIPPAGE"
+	// TradeTLSKeyKey is the path of the the TLS key for the Trade interface
+	TradeTLSKeyKey = "TRADE_TLS_KEY"
+	// TradeTLSCertKey is the path of the the TLS certificate for the Trade interface
+	TradeTLSCertKey = "TRADE_TLS_CERT"
+	// EnableProfilerKey enables profiler that can be used to investigate performance issues
+	EnableProfilerKey = "ENABLE_PROFILER"
+	// StatsIntervalKey defines interval for printing basic tdex statistics
+	StatsIntervalKey = "STATS_INTERVAL"
+	// NoMacaroonsKey is used to start the daemon without using macaroons auth
+	// service.
+	NoMacaroonsKey = "NO_MACAROONS"
+	// NoOperatorTlsKey is used to start the daemon without using TLS for the operator service
+	NoOperatorTlsKey = "NO_OPERATOR_TLS"
+	// OperatorExtraIP is used to add an extra ip address to the self-signed TLS
+	// certificate for the Operator gRPC interface.
+	OperatorExtraIPKey = "OPERATOR_EXTRA_IP"
+	// OperatorExtraDomain is used to add an extra domain to the self signed TLS
+	// certificate for the Operator gRPC interface. This is useful to add the
+	// onion endpoint in case the daemon is served as a TOR hidden service for
+	// example.
+	OperatorExtraDomainKey = "OPERATOR_EXTRA_DOMAIN"
+	// WalletUnlockPasswordFile defines full path to a file  that contains the
+	//password for unlocking the wallet, if provided wallet will be unlocked
+	//automatically
+	WalletUnlockPasswordFile = "WALLET_UNLOCK_PASSWORD_FILE"
+	// ConnectAddrKey is the address <host:port> of the tdex-daemon service to connect to, ie. myservice:9000
+	ConnectAddrKey = "CONNECT_ADDR"
+	// ConnectUrlProtoKey is the http/https protocol of the tdex-daemon service to connect to
+	// used by tdexconnect to create connection url
+	ConnectProtoKey = "CONNECT_PROTO"
+	// OceanWalletAddrKey is the address for connecting to the ocean wallet.
+	OceanWalletAddrKey = "WALLET_ADDR"
+	// DBTypeKey is used to switch database type between those supported
+	DBTypeKey = "DB_TYPE"
+
+	DbLocation        = "db"
+	TLSLocation       = "tls"
+	MacaroonsLocation = "macaroons"
+	ProfilerLocation  = "stats"
+
+	minPercentageFee = 0.01
+	maxPercentageFee = float64(99.99)
+
+	httpsProtocol = "https"
+)
+
+var vip *viper.Viper
+var defaultDatadir = btcutil.AppDataDir("tdex-daemon", false)
+
+func init() {
```

Comment:
Personally, if i am not forced, I am done using init()  :)
I think it only inserts unpredictable/uncontrolled behaviour

Reply:
will move to InitConfig func

Reply:
done.